File: leetcode-solutions/0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if (nums[i]+nums[j]) == target:
                    return [i,j]
                    break
            # print(i)
            # if ((nums[i]+nums[i+1])%3) == 0 and i<len(nums):
            #     # print(nums[i],nums[i+1])
            #     return [i,i+1]
            #     break

File: leetcode-solutions/0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        return two_sum(nums, target);

    }

    public static int [] two_sum(int [] arr, int n){
		int len = arr.length;
		int [] a = new int [2];
		for (int i = 0; i<len; i++) {
			for (int j = i+1; j<len; j++) {
				if(arr[i]+arr[j]==n) {
					a[0] = i;
					a[1] = j;
					
				}
			}
		}
		return a;
	}
}
